arch/arm64/boot/dts/qcom/sm7150-xiaomi-tucana.dts

```diff
@@ -1,23 +1,35 @@
 // SPDX-License-Identifier: BSD-3-Clause
 /*
  * Copyright (c) 2024, Danila Tikhonov <danila@jiaxyga.com>
+ * Copyright (c) 2024, Alexander Baransky <alexander.baranskiy@yandex.ru>
```

Comment:
Add the Signed-off-by tag in your commit message if you're not staying anonymous.